### PR TITLE
Hides tickets section from public/index if details are not present

### LINKED ISSUE #489: Hide ticket section if no details are present
**I'm submitting a ...**  (check one with "x")
- [x] bug report
- [ ] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

**Current behavior:**
screen shot 2017-07-17 at 5 33 44 pm

The tickets section is visible if ticketing is enabled but there are no tickets for the event.

**Expected behavior:**
The public/index route must not have tickets section if there is no ticket detail available

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)

Hides the tickets section if not ticket details is available for the event.
Hides if :
- No tickets are available and no ticket URL is available

Deployment Link: https://open-event-frontend-app.herokuapp.com
Fixes #489

### CODE REVIEW

File: app/templates/public/index.hbs [hunk 1]
```diff
@@ -3,35 +3,37 @@
...
-          {{public/ticket-list ticketUrl=model.event.ticketUrl isTicketingEnabled=model.event.isTicketingEnabled tickets=model.tickets}}
-        {{else}}
-          <div class="ui grid">
-            <div class="ui row">
-              <div class="column">
-                <a target="_blank" rel="noopener noreferrer" href="{{ticketUrl}}" class="ui right labeled blue icon button">
-                  <i class="ticket icon"></i>
-                  {{t 'Order tickets'}}
-                </a>
+  {{#if (or ticketUrl (gt model.tickets.length 0))}}
```
Comment (niranjan94) [inline app/templates/public/index.hbs]: You can just do `{{#if (or ticketUrl model.tickets)}}` length of 0 is considered to be _falsy_ by handlebars

Comment (geekyd) [inline app/templates/public/index.hbs]: Done!

File: app/templates/public/index.hbs [hunk 2]
```diff
@@ -3,35 +3,37 @@
...
-          {{public/ticket-list ticketUrl=model.event.ticketUrl isTicketingEnabled=model.event.isTicketingEnabled tickets=model.tickets}}
-        {{else}}
-          <div class="ui grid">
-            <div class="ui row">
-              <div class="column">
-                <a target="_blank" rel="noopener noreferrer" href="{{ticketUrl}}" class="ui right labeled blue icon button">
-                  <i class="ticket icon"></i>
-                  {{t 'Order tickets'}}
-                </a>
+  {{#if (or ticketUrl model.tickets.length)}}
```
Comment (niranjan94) [inline app/templates/public/index.hbs]: Just `model.tickets` would do. Empty arrays are considered falsy by handlebars

**Comment by geekyd** [comment]
@fossasia/open-event-frontend Please review again. Thanks

**Comment by CosmicCoder96** [comment]
@geekyd Can you please check so as to what might have failed the build :)

**Comment by geekyd** [comment]
@CosmicCoder96 The problem was at travis's end. Now all the tests are passing. Please review. Thanks

**Pull request merged**